src/averageAges.js

```diff
@@ -55,7 +70,15 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const womenAges = people.filter(human => onlyWithSon
+    ? ((human.sex === 'm') === onlyWithSon)
```

Comment:
why do you need this here? onlyWithSon is just a boolean
`(human.sex === 'm') === onlyWithSon`